README.md

```diff
@@ -0,0 +1,174 @@
+# Book vehicle rental backend
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:question: FAQ](#faq)
+- [:memo: License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 [Book vehicle rental backend] <a name="about-project"></a>
+
+**[Book vehicle rental backend]** is a backend rails api that provides service to the front-end react app to fetch data for vehile rentals.
+
+## [Link to the front-end repo](https://github.com/HERMON-1995/Book_vehicle_rental-frontend/tree/development)
+
+Below is the link to the database diagram
+
+<img src="app/images/Vehicle appointment database image 2.jpg" alt="Vehicle appointment diagram" />
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a>https://rubyonrails.org/</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Render vehicles]**
+- **[Add vehicles]**
+- **[Delete vehicles]**
+- **[Reserve vehicles]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Install Ruby
+- Install Rails
+- Install Postgresql
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Book_vehicle_rental-backend
+  git clone https://github.com/HERMON-1995/Book_vehicle_rental-backend.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Book_vehicle_rental-backend
+  bundle install
+```
+
+```sh
+  gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server -p 3001
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rubocop --color
+```
```

Comment:
- [ ] Kindly, correct this command to your your tests.  You need to use rspec:

```bash
rspec ./spec/controllers
rspec ./spec/requests
rspec ./spec/model
```

That way, all your tests will pass. 👍

Reply:
Thanks for the insight, test cases added.